### PR TITLE
[16.0] [MIG] analytic_partner

### PR DESCRIPTION
**Pull request by gaelTorrecillas**
Hi,

I fork this pr https://github.com/OCA/account-analytic/pull/609 and I add all your recommendations.

### CODE REVIEW

**Comment by flotho** [comment]
Hi @gaelTorrecillas 
thanks for this PR.
First of all, could you please change  the PR name and  the commit names with a rebase to follow the guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#git
Then for your re commit issue, could you run `pre-commit run --all-files` and commit the result in a dedicated commit for lint

**Comment by flotho** [comment]
Hi @gaelTorrecillas 
Thanks for this PR .
Codecov is sometimes weird as I discovered with @pedrobaeza 
My proposal is that you rename your PR and  commits following the guidelines and then we'll be able to make a proper review and maybe we'll have better ideas for the codecov regression.

Regards

**Comment by flotho** [comment]
Hi @gaelTorrecillas thanks, could you change the PR name please?

File: analytic_partner/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {
     "name": "Link analytic items and partner",
     "summary": "Search and group analytic entries by partner",
-    "version": "14.0.1.0.0",
+    "version": "16.0.0.1.0",
```
Comment (peluko00) [inline analytic_partner/__manifest__.py]: `16.0.1.0.0`

**Review by peluko00 — CHANGES_REQUESTED** [review]

File: analytic_partner/__manifest__.py (hunk 1 above)
Comment (peluko00) [reply analytic_partner/__manifest__.py]: @gaelTorrecillas Can you squash all migrations commits into one and squash administrive commits.

Comment (gaelTorrecillas) [reply analytic_partner/__manifest__.py]: hello, thanks you for your review.

I'll squash my future commit no problem.

regards,

**Review by DorianMAG — APPROVED** [review]
Code review
Seems to be good
Thx for this work

**Review by JulienMartinez — APPROVED** [review]
Code review
LTGM

**Comment by peluko00** [comment]
Hi @gaelTorrecillas, where is the pre-commit commit before migration commit?

**Comment by flotho** [comment]
> Hi @gaelTorrecillas, where is the pre-commit commit before migration commit?

Holidays for @gaelTorrecillas more to come in a few weeks

**Review by dreispt — APPROVED** [review]

**Pull request merged**